<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDispgraphic</class>
 <widget class="QWidget" name="QDispgraphic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Graphic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBox_DevIndex">
       <property name="text">
        <string>Device ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_DevIndex"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_StartEndDate">
       <property name="text">
        <string>DateTime</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_Start">
       <property name="displayFormat">
        <string>yyyy年M月d日</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;-&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_End">
       <property name="displayFormat">
        <string>yyyy年M月d日</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_PipeIndex">
       <property name="text">
        <string>Pipe ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_PipeIndexStart"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;-&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_PipeIndexEnd"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Ch4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AbsPressure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SfcPressure</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Temperature</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CO</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CO2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>O2</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_print">
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QwtPlot" name="qwtPlot">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="canvasBackground">
      <brush brushstyle="Dense6Pattern">
       <color alpha="255">
        <red>212</red>
        <green>208</green>
        <blue>200</blue>
       </color>
      </brush>
     </property>
     <property name="autoReplot">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
